concurrentaccess_test.go

```diff
@@ -0,0 +1,119 @@
+//   Copyright © 2015-2017 Ivan A Kostko (github.com/ivan-kostko; github.com/gopot)
+
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+
+//       http://www.apache.org/licenses/LICENSE-2.0
+
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+
+// +build race
+
+package concurrentmap_test
+
+import (
+	. "concurrent-map"
+	"reflect"
+	"sync"
+	"testing"
+)
+
+func TestConcurrentOperations(t *testing.T) {
+	testCases := []struct {
+		TestAlias   string
+		Iterations  int
+		Routines    int
```

Comment:
Would you mind to decouple `Iterations` and `Routines` into a seperate slice (maybe `testEnviroment`)? This way could test different Iterations/Routines combinations with all TestCases without copying code.